cfme/common/vm.py

```diff
@@ -641,8 +641,11 @@ def _looking_for_state_change():
                 self.load_details(refresh=True)
                 return self.get_detail(properties=detail_t) == desired_state
             else:
+                key = version.pick({
```

Comment:
**Required** Use self.appliance.version here instead

Reply:
This fix is no longer needed, "state" will work with 5.9.0.9.
I used version.pick() because it is used across the file and I wanted  to be consistent. self.appliance.version is not used in this file.